vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -25,7 +25,7 @@
 from vizro.models.types import MultiValueType, SelectorType
 
 if TYPE_CHECKING:
-    from vizro.models import Page
+    pass
```

Comment:
Ah yes, this can be removed now! We previously needed it for the circular import errors, but given that @petar-qb refactored it, we don't need the import of the Page model anywhere